webapp/src/components/jira_field.jsx

```diff
@@ -175,7 +175,7 @@ export default class JiraField extends React.Component {
                 }
 
                 const onChange = (id, val) => {
-                    const newValue = val.map((v) => ({id: v}));
+                    const newValue = val ? val.map((v) => ({id: v})) : [];
```

Comment:
Is optional chaining available in this repo?

```suggestion
                    const newValue = val?.map((v) => ({id: v})) || [];
```

Reply:
@mickmister no optional chaining is not available in this repo.

Reply:
This seems a bit error prone to not check if this is an array first, but the "field type" is an array so we can expect this type